src/transformStateWithClones.js

```diff
@@ -8,6 +8,33 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+
+  const stateCopy = { ...state };
+  const arrOfActions = [];
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(stateCopy, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          if (stateCopy[key]) {
+            delete stateCopy[key];
+          }
+        }
+        break;
+
+      default:
+        for (const key in stateCopy) {
+          delete stateCopy[key];
+        }
```

Comment:
there should be third case` clear` and then have `default` case , do not include case` clear` in default

Reply:
thanks